src/featureconverter.js

```diff
@@ -295,21 +309,39 @@ olcs.FeatureConverter.prototype.olLineStringGeometryToCesium =
     material: this.olStyleToCesium(feature, olStyle, true)
   });
 
-  // Handle both color and width
-  var outlineGeometry = new Cesium.PolylineGeometry({
-    // always update Cesium externs before adding a property
-    positions: positions,
-    width: this.extractLineWidthFromOlStyle(olStyle),
-    vertexFormat: appearance.vertexFormat
-  });
+  var heightReference = this.getHeightReference(layer, feature, olGeometry);
+
+  var outlinePrimitive;
+
+  if (heightReference == Cesium.HeightReference.CLAMP_TO_GROUND) {
+    // Handle both color and width
+    var outlineGeometry = new Cesium.CorridorGeometry({
+      // always update Cesium externs before adding a property
+      positions: positions,
+      width: this.extractLineWidthFromOlStyle(olStyle) * 10000,
```

Comment:
The conversion factor can not be static, see https://github.com/openlayers/ol3-cesium/pull/243#issuecomment-140387081